src/SDKs/CognitiveServices/dataPlane/AnomalyDetector/AnomalyDetector/Microsoft.Azure.CognitiveServices.AnomalyDetector.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.AnomalyDetector</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services AnomalyDetector APIs.</Description>
```

Comment:
Please remove `Description`

Reply:
But why?

Reply:
The description for nuget package should be described in the `PackageReleaseNotes` section per our guidelines, since you already have that in the csproj, `Description` is not necessary